### DISCUSSION TITLE
Feed Errors and Warnings

### DISCUSSION THREAD

**Discussion by Snipe3000** [nzbgetcom/nzbget#464] [Q&A]
I seem to be getting a bunch of errors and warning because of my feeds, but there is not enough information for me to troubleshoot.

```
warning	URL #TV failed: HTTP/1.1 520
error        Download #TV failed
error        Could not read from TLS-Socket: Connection closed by remote host
error	Failed to parse rss feed #TV
error	Error parsing rss feed #TV: xmlParseStartTag: invalid element name
error	Feed file /volume1/NZBget/tmp/feed-72.tmp kept for troubleshooting (will be deleted on next successful feed fetch)
warning	URL #TV failed: HTTP/1.1 503 Service Unavailable
```

I'm seeing this for almost all of my feeds and it spams the logs. I copy and paste the feed URLs into the web browser and they all work, so I have no idea what the problem is.
This NZBget is loaded on a synology box.

**Comment by dnzbk** [comment]
What version of NZBGet? 
Could you share one of these feeds for testing? 
denis@nzbget.com
